tap_invoiced/schemas/plans.json

```diff
@@ -0,0 +1,106 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "properties": {
+    "amount": {
+      "type": [
+        "null",
+        "number"
+      ]
+    },
+    "catalog_item": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "currency": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "description": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "interval": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "interval_count": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "metadata": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {}
+    },
+    "name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "notes": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "object": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "pricing_mode": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "quantity_type": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "tiers": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {}
+    },
```

Comment:
@jaredtking thanks so much for the help here! I've updated `tiers` to match the same as `metadata` property is being handled across other schemas.